packages/@react-spectrum/menu/src/ActionMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import {ActionButton} from '@react-spectrum/button';
+import {filterDOMProps} from '@react-aria/utils';
+import {FocusableRef} from '@react-types/shared';
+import {Menu} from './Menu';
+import {MenuTrigger} from './MenuTrigger';
+import More from '@spectrum-icons/workflow/More';
+import React from 'react';
+import {SpectrumActionMenuProps} from '@react-types/menu';
+
+function ActionMenu<T extends object>(props: SpectrumActionMenuProps<T>, ref: FocusableRef<HTMLButtonElement>) {
+  let buttonProps = filterDOMProps(props);
+  if (buttonProps['aria-label'] === undefined) {
+    buttonProps['aria-label'] = 'More actions';
```

Comment:
we'll need to get this localized, it'll need to go into an `intl` directory, you can find other instances of this in React Spectrum

Reply:
i extracted it and added l10n for US and DE. What service do you use for translation google-translate, an adobe-internal system?